R/utils.R

```diff
@@ -100,6 +100,33 @@ r_files <- function(path) {
   sort_c(dir(file.path(path, "R"), "[.Rr]$", full.names = TRUE))
 }
 
+ignore_files <- function(path, rfiles) {
+    rbuildignore <- file.path(path, ".Rbuildignore")
+    if (!file.exists(rbuildignore)) {
+        rfiles
```

Comment:
It would be cleaner to `return(rfiles)` here so that the big block below doesn't need to be indented